error: unused parameter 'name' [-Werror,-Wunused-parameter]

```
/usr/ports/misc/astc-encoder/work/astc-encoder-5.1.0/Source/astcenccli_platform_dependents.cpp:192:14: error: unused parameter 'name' [-Werror,-Wunused-parameter]
  192 |         const char* name
      |                     ^
1 error generated.
```

--- a/Source/astcenccli_platform_dependents.cpp
+++ b/Source/astcenccli_platform_dependents.cpp
@@ -196,6 +196,8 @@ void set_thread_name(
 	pthread_setname_np(pthread_self(), name);
 #elif defined(__APPLE__)
 	pthread_setname_np(name);
+#else
+	(void)name;
 #endif
 }
 

